### PR TITLE
SVA-403 Adds authentication API

### PR DESCRIPTION
**Pull request by roryf**
# Description

https://sta2020.atlassian.net/browse/SVA-403

Adds `/auth` endpoints to support Cognito-based authentication in the app.

Requires config updates to set Cognito env vars, depends on setting up Cognito correctly.

## Type of change

Please delete options that are not relevant

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New Feature (non-breaking change which adds functionality)
- [ ] Breaking change(fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Testing locally

Won't currently work locally or otherwise...

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have removed any unnecessary comments or console logging
- [ ] I have made corresponding changes to the documentation (if required)
- [ ] I have addressed accessibility, if needed
- [ ] I have followed best practices, e.g. NativeBase approaches and theming
- [ ] I have checked the app in dark mode, if making front-end design changes
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have updated the version numbers in `package.json` files in the [app](DEPLOYMENT.md#app-deployment) and/or [api](DEPLOYMENT.md#api-deployment-on-aws) directories as needed

### CODE REVIEW

File: api/routes/v1.1/index.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
-const express = require('express');
+const express = require("express");
```
Comment (roryf) [inline api/routes/v1.1/index.js:1]: Prettier decided everything needed double quotes, will update once #198 lands

**Review by PammyLo — APPROVED** [review]
Many thanks for getting this started @roryf 
Can't test locally but doesn't break anything and code looks good :)

**Review by DJMF2015 — APPROVED** [review]
@roryf i have taken a look over the code changes and run the unit tests (all which pass). I can't see anything that stands out currenlty that requires needing changed. I appreciate there is further work to do joining up Cognito going forward. It looks good work and thanks for doing this. Happy to approve.

**Comment by roryf** [comment]
Merging now as doesn't break existing functionality, even though new endpoints will currently fail.

**Pull request merged**